src/Security/Security.php

```diff
@@ -906,32 +962,21 @@ public function getTemplatesFor($action)
      */
     public static function findAnAdministrator()
     {
-        // coupling to subsites module
-        $origSubsite = null;
-        if (is_callable('Subsite::changeSubsite')) {
-            $origSubsite = Subsite::currentSubsiteID();
-            Subsite::changeSubsite(0);
-        }
+        static::singleton()->extend('beforeFindAdministrator');
```

Comment:
Subsite obviously needed to be _not_ a dependency in core anymore.

In discussion with @Cheddam  and having a look around at the usage of the code, I figured this is the cleanest way to do it, and let Subsite hook into this method when needed.